src/graph_export/graph_export.cpp

```diff
@@ -105,25 +105,187 @@ static std::string createTextGenerationGraphTemplate(const GraphSettingsImpl& gr
     return oss.str();
 }
 
+static std::string createRerankSubconfigTemplate(const RerankGraphSettingsImpl& graphSettings) {
+    std::ostringstream oss;
+    // clang-format off
+    oss << R"(
+    {
+        "model_config_list": [
+            { "config":
+                {
+                    "name": ")" << graphSettings.modelName << R"(_tokenizer_model",
```

Comment:
at which point model name/target device is being validated so it won't create non-json(special chars etc)?

Reply:
Maybe we should add test graph.pbtxt parsing after graph is created ?